importer/SeBbrSv.py

```diff
@@ -302,8 +312,44 @@ def exists_with_monument_article(self, language):
         """Set language of Wikipedia to use to match articles."""
         return super().exists_with_monument_article("sv")
 
+    def get_wditems_bbr_links(self):
+        """
+        Get BBR links of associated WD item.
+
+        If the data object is associated
+        with a WD item, check if it has P1260
+        of the type raa/bbr(a) and get their
+        values.
+        """
+        result = []
+        associated_item = self.wd_item["wd-item"]
+        if associated_item:
+            heritage_links = utils.get_value_of_property(
+                associated_item, "P1260", self.repo)
+            for link in heritage_links:
+                if link.startswith("raa/bbr"):
+                    result.append(link)
+            return result
+
+    def check_wd_item(self):
+        """
+        Detect BBR conflict in associated WD item.
+
+        Get any P1260 of associated WD item.
+        If it's a BBR ID that's different
+        from the one in the data object,
+        remove the association. Data will be uploaded
+        to a newly created WD item instead.
+        """
+        bbr_url_on_wd = self.get_wditems_bbr_links()
+        if bbr_url_on_wd:
+            for url in bbr_url_on_wd:
```

Comment:
you can do this as `if any(url != self.bbr_link for url in bbr_url_on_wd):` the benefit being that it stops as soon as it encounters a hit.